extensions/wikia/ArticleVideo/ArticleVideoContext.class.php

```diff
@@ -5,6 +5,22 @@ class ArticleVideoContext {
 	const ARTICLE_VIDEO_ERROR_MESSAGE = 'JWPlayer: Could not find mediaId in article-video service';
 	const JWPLAYER_API_ERROR_MESSAGE = 'JWPlayer: Could not find enough playback info in JW API to play the video';
 
+	/**
+	 * to get playlist for given article: $playlist = self::RECOMMENDED_VIDEO_MAPPINGS[$cityId][$articleId]
+	 */
+	const RECOMMENDED_VIDEO_MAPPINGS_POZ_DEV = [
+		// harrypotter Knight_Bus
+		509 => [
+			509 => '1Dw2sjAj',
+		]
+	];
+
+	const RECOMMENDED_VIDEO_MAPPINGS_PROD = [
```

Comment:
We have to remember to add correct video mappings/remove unnecessary one.

Reply:
👍